doc/concepts/mechanisms.rst

```diff
@@ -76,9 +76,55 @@ Two catalogues are provided that collect mechanisms associated with specific pro
 * *bbp* For models published by the Blue Brain Project (BBP).
 * *allen* For models published on the Allen Brain Atlas Database.
 
-Further catalogues can be added by extending the list of built-in catalogues in
-the arbor source tree or by compiling a dynamically loadable catalogue
-(:ref:`extending catalogues <extending-catalogues>`).
+.. _mechanisms_dynamic:
+
+Adding Catalogues to Arbor
+''''''''''''''''''''''''''
+
+.. Note::
+
+   If you are coming from NEURON this is the equivalent of ``nrnivmodl``.
+
+This will produce a catalogue loadable at runtime by calling ``load_catalogue``
+with a filename in both C++ and Python. The steps are
+
+1. Prepare a directory containing your NMODL files (.mod suffixes required)
+2. Call ``build-catalogue`` from the ``scripts`` directory
```

Comment:
I would prefer to rename build-catalogue to arbor-build-catalogue and install it to $INSTALL_PREFIX/bin

Reply:
`build-catalogue` is already being installed into `$PREFIX/bin`, the docs are outdated here, thanks